crates/indexer/migrations/2022-09-22-185845_token_offers/up.sql

```diff
@@ -0,0 +1,160 @@
+-- Your SQL goes here
+ALTER TABLE current_token_ownerships
+ADD COLUMN collection_data_id_hash VARCHAR(64) NOT NULL DEFAULT '',
+  ADD COLUMN table_type TEXT NOT NULL DEFAULT '';
+ALTER TABLE current_token_datas
+ADD COLUMN collection_data_id_hash VARCHAR(64) NOT NULL DEFAULT '';
+ALTER TABLE token_datas
+ADD COLUMN collection_data_id_hash VARCHAR(64) NOT NULL DEFAULT '';
+ALTER TABLE tokens
+ADD COLUMN collection_data_id_hash VARCHAR(64) NOT NULL DEFAULT '';
+ALTER TABLE token_ownerships
+ADD COLUMN collection_data_id_hash VARCHAR(64) NOT NULL DEFAULT '';
```

Comment:
nullable pls

Reply:
I'll remove the default since the backfill is already done.